cms/models/fields.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, slotname, default_width=None, actions=None, **kwargs):
         # We hard-code the `to` argument for ForeignKey.__init__
         # since a PlaceholderField can only be a ForeignKey to a Placeholder
         kwargs['to'] = 'cms.Placeholder'
+        kwargs['on_delete'] = models.CASCADE
```

Comment:
Setting this here means user still needs to set `on_delete` in the migrations no?
If so, then this would silence the warning from Django and thus user won't know to update the migration files for model that uses this field.

Reply:
No, this means that `my_placeholder = PlaceholderField('placeholder_name')` still works as documented. `models.CASCADE` is always used and can't be overridden. If there's a use case for overriding it, then this could be `kwargs.setdefault('on_delete', models.CASCADE)`.